code/game/machinery/atmoalter/portable_atmospherics.dm

```diff
@@ -110,21 +110,21 @@
 	else if(isWrench(W) && !panel_open)
 		if(connected_port)
 			disconnect()
-			to_chat(user, "<span class='notice'>You disconnect \the [src] from the port.</span>")
+			to_chat(user, SPAN_NOTICE("You disconnect \the [src] from the port."))
 			update_icon()
 			return
 		else
 			var/obj/machinery/atmospherics/portables_connector/possible_port = locate(/obj/machinery/atmospherics/portables_connector/) in loc
 			if(possible_port)
 				if(connect(possible_port))
-					to_chat(user, "<span class='notice'>You connect \the [src] to the port.</span>")
+					to_chat(user, SPAN_NOTICE("You connect \the [src] to the port."))
 					update_icon()
 					return
 				else
-					to_chat(user, "<span class='notice'>\The [src] failed to connect to the port.</span>")
+					to_chat(user, SPAN_NOTICE("\The [src] failed to connect to the port."))
 					return
 			else
-				to_chat(user, "<span class='notice'>Nothing happens.</span>")
+				to_chat(user, SPAN_NOTICE("Nothing happens."))
```

Comment:
```suggestion
```
This should probably silently fail, since `..()` *could* implement functionality; then you'd get `Nothing happens.` followed by... something happening.